src/main/java/application/Routes.java

```diff
@@ -33,7 +34,11 @@
 
         router.get("/redirect", Handlers.Redirect);
 
-        router.get("/", Handlers.SimpleGet);
+        router.get("/http://www.ctabustracker.com/bustime/api/v2/getroutes", Handlers.Proxy);
+        router.get("/http://www.ctabustracker.com/bustime/api/v2/getpredictions", Handlers.Proxy);
+        router.get("/http://www.ctabustracker.com/bustime/api/v2/getpatterns", Handlers.Proxy);
+        router.get("/http://www.ctabustracker.com/bustime/api/v2/getstops", Handlers.Proxy);
+        router.get("/http://www.ctabustracker.com/bustime/api/v2/getdirections", Handlers.Proxy);
```

Comment:
would it be worth masking the proxy a bit? Thinking you could perhaps simplify the url by doing `router.get("/getdirections")` and having your server encapsulate the knowledge of `http://www.ctabustracker.com/bustime/api/v2`.